src/main/java/emu/grasscutter/command/commands/TeamCommand.java

```diff
@@ -0,0 +1,73 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.Grasscutter;
+import emu.grasscutter.command.Command;
+import emu.grasscutter.command.CommandHandler;
+import emu.grasscutter.game.avatar.Avatar;
+import emu.grasscutter.game.player.Player;
+import emu.grasscutter.server.packet.send.PacketChangeMpTeamAvatarRsp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static emu.grasscutter.utils.Language.translate;
+
+@Command(label = "team", usage = "team [add/remove] [AvatarID]",
+        description = "commands.join.description", permission = "player.join"+"player.remove")
+public class TeamCommand implements CommandHandler {
+
+    @Override
+    public void execute(Player sender, Player targetPlayer, List<String> args) {
+        int branch = 0;
+        switch(args.get(0)){
```

Comment:
No length check on `args`, this will throw when `/team` is called without arguments